### ISSUE TITLE
September 2020 Changelog

### ISSUE THREAD

**Issue by carolyn-shannon** [pantheon-systems/documentation#5900]
For updates occurring in August, 2020. 
When entering product updates, please use the following format:

```
### Subject (i.e WP-CLI Upgrades)
Short description, like: WP-Cl has been upgraded to [1.5.0](https://make.wordpress.org/cli/2018/01/31/version-1-5-0-released/).

Priority: Low

**Comment by greg-1-anderson** [comment]
### WordPress 5.5

WordPress 5.5 has been released on the Pantheon platform. This version of WordPress introduces a new method, [wp_get_environment_type()](https://make.wordpress.org/core/2020/07/24/new-wp_get_environment_type-function-in-wordpress-5-5/), which returns whether your site is running on a live, test or development environment. Pantheon supports this method, so it will return the correct value on any site that has upgraded to WordPress 5.5.

**Comment by pwtyler** [comment]
## WordPress 5.5.1

WordPress 5.5.1 is a maintenance release with multiple bug fixes and enhancements, including fixes for changes introduced in WordPress 5.5. For the full list of bug fixes and enhancements added in 5.5.1, see https://wordpress.org/news/2020/09/wordpress-5-5-1-maintenance-release/.

**Comment by carolyn-shannon** [comment]
WP 5.5.1 update will appear in Oct changelog.
